libs/application/template-api-modules/src/lib/modules/shared/api/national-registry/national-registry.service.ts

```diff
@@ -31,11 +31,34 @@ export class NationalRegistryService extends BaseTemplateApiService {
     auth,
     params,
   }: TemplateApiModuleActionProps<NationalRegistryParameters>): Promise<NationalRegistryIndividual | null> {
+    const individuals: Array<NationalRegistryIndividual | null> = []
     const result = await this.getIndividual(auth.nationalId)
+    individuals.push(result)
+
+    if (params?.allowDomicilePassOnChild) {
```

Comment:
Are we not checking for more than just domicile?  `includeChildrenInValidation`  as suggestion ?